video_emotion/emotionTagger/t/test_face_emotion_extraction.py

```diff
@@ -0,0 +1,77 @@
+from video_emotion.emotionTagger.face_emotion_extraction import classify_face
+from video_emotion.emotionTagger.face_emotion_extraction import get_labels
+from video_emotion.emotionTagger.face_emotion_extraction import preprocess_input
+import numpy as np
+import cv2
+import os
+
+def test_angry_face():
+    dirname = os.path.abspath(os.path.dirname(__file__))
+    filename = os.path.join(
+        dirname,
+        "test_face_emotion_extraction_data/angry1.png")
+    im = cv2.imread(filename)
```

Comment:
'im' variable name?

Reply:
Probably short for image. Agree that img would make more sense.